### PR TITLE
Updates to the lib/version

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

### CODE REVIEW

File: test/gen-loadastic.go [hunk 1]
```diff
@@ -9,8 +9,9 @@ import (
 	"sync"
 	"time"
 
+	"github.com/google/uuid"
 	"github.com/slinkydeveloper/loadastic/common"
-	vegeta "github.com/slinkydeveloper/vegeta/lib"
+	vegeta "github.com/tsenart/vegeta/v12/lib"
```
Comment (matzew) [inline test/gen-loadastic.go:14]: Not sure that's really correct, but I removed your fork....

File: templates/loadastic.go [hunk 2]
```diff
@@ -4,13 +4,14 @@ package templates
...
-	"github.com/slinkydeveloper/loadastic/common"
-	vegeta "github.com/tsenart/vegeta/lib"
 	"runtime"
 	"sync"
 	"time"
+
+	"github.com/cheekybits/genny/generic"
+	"github.com/google/uuid"
+	"github.com/slinkydeveloper/loadastic/common"
+	vegeta "github.com/tsenart/vegeta/v12/lib"
```
Comment (matzew) [inline templates/loadastic.go:14]: sorry for the noise, but the editor did it... I did not even notie

**Comment by matzew** [comment]
the `go.sum` seems a bit weird ... ?!?

**Comment by slinkydeveloper** [comment]
Does everything compiles? Did you tried `go vet ./...` to check?

> the go.sum seems a bit weird ... ?!?

It might be, It's a while that I don't touch this code...

**Comment by matzew** [comment]
let me see...

**Comment by matzew** [comment]
@slinkydeveloper looks like the mock now needs to have one more methid:

```
vet: test/loadastic_test.go:39:30: cannot use &sender (value of type *mockSender) as Sender value in argument to NewLoadastic: missing method InitializeWorker
```

but yeah - I did not change any of that :-)

**Comment by slinkydeveloper** [comment]
Can you fix the mock? :)

**Comment by matzew** [comment]
@slinkydeveloper added a bit

**Pull request merged**